### PR TITLE
MineMezzo prototype

### LINKED ISSUE #67: make a working mezzo/module prototype
Should have:

 * a mezzo
 * a gameview
 * some minigames
    * at least one all-React
    * at least one non-React

Should obey as much of the module/mezzo contract as possible.  Flesh it out as necessary.  Fun or interesting gameplay is not part of this ticket; the point is to prototype the architecture.

### PR DESCRIPTION
**Pull request by jholman**
Prototype/stub for a working mezzo/gameview/minigames.

Resolves #67 

Contains a mezzo, a gameview, and two minigames (one react, one non-react).

### CODE REVIEW

File: src/modules/non_react_template.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+//
+// This is some non-react code that makes use of the stuff above
+//
+////////
+
+
+const settableWhitelistSet = new Set(['ctx', 'hsl']);
+const initableWhitelistSet = new Set(['ctx', 'hsl']);
+
+function multiAssign(that, assignments, whitelist) {
```
Comment (garrettgsb) [inline src/modules/non_react_template.js]: JEREMY YOU'RE DOING IT AGAIN

File: src/modules/non_react_template.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    if (whitelist.has(argname)) {
+      that[argname] = assignments[argname];
+    }
+  }
+}
+
+class ContainedThing {
+  constructor(args) {
+    this.init(args);
+    window.aag = this;      // debug cheating!
```
Comment (garrettgsb) [inline src/modules/non_react_template.js]: If this is a template, aren't we going to do a ton of overwriting for anyone using this?

File: src/modules/non_react_template.js (hunk 1 above)
Comment (jholman) [reply src/modules/non_react_template.js]: truest comment is best comment

File: src/modules/non_react_template.js (hunk 2 above)
Comment (jholman) [reply src/modules/non_react_template.js]: I mean, whatever.  I already deviated from it in my very first use of it.  This is me caring about this: 😐

File: src/lib/rpot.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import autobind from 'autobind-decorator';
+
+class RelentlessPassageOfTime {
```
[the remark this answers is not in the document]
Comment (jholman) [reply src/lib/rpot.js]: Core concept is identical.  The majority of the details are changed.  Different signature for the `tick()` it calls on the subscribers, different contructor, no DI, there's that scraps data structure, etc.

**Pull request merged**